core/aws-core/src/main/java/software/amazon/awssdk/awscore/AwsServiceClientConfiguration.java

```diff
@@ -90,19 +92,6 @@ protected BuilderImpl(AwsServiceClientConfiguration awsServiceClientConfiguratio
             this.region = awsServiceClientConfiguration.region();
         }
 
-
-        @Override
-        public Builder overrideConfiguration(ClientOverrideConfiguration clientOverrideConfiguration) {
-            this.overrideConfiguration = clientOverrideConfiguration;
-            return this;
-        }
```

Comment:
Shouldn't we have this on the interface?

Reply:
added in Builder interface